applications/tari_mining_node/src/main.rs

```diff
@@ -113,6 +122,7 @@ async fn mining_cycle(
     node_conn: &mut BaseNodeClient<Channel>,
     wallet_conn: &mut WalletClient<Channel>,
     config: &MinerConfig,
+    bootstrap: &ConfigBootstrap,
```

Comment:
I don't like the reuse of ConfigBootstrap here. It's much better to create a new struct that is specific to this app only